package.json

```diff
@@ -10,8 +10,8 @@
     "lint": "ionic-app-scripts lint",
     "ionic:build": "ionic-app-scripts build",
     "ionic:serve": "ionic-app-scripts serve",
-    "ionic:android": "ionic cordova run android --emulator --no-fetch",
```

Comment:
Should we keep the --no-fetch flag to avoid losing sym links when using local sdk version

Reply:
What version of cordova are you using? --no-fetch is no longer supported in cordova 8 which is what is being used by default.

Reply:
Yeah I am using version 8, good catch, we might want to make an update to the docs, to make note of it for anyone developing the SDK, or are we only supporting cordova 8?